tests/test_docker.sh

```diff
@@ -19,3 +19,6 @@ docker-compose -f local.yml run django python manage.py test
 
 # return non-zero status code if there are migrations that have not been created
 docker-compose -f local.yml run django python manage.py makemigrations --dry-run --check || { echo "ERROR: there were changes in the models, but migration listed above have not been created and are not saved in version control"; exit 1; }
+
+# Test support for translations
+docker-compose -f local.yml run django python manage.py makemessages
```

Comment:
I'd make it `docker-compose -f local.yml run --rm django python manage.py makemessages`.